### DISCUSSION TITLE
Ingesting new data on data cube

### DISCUSSION THREAD

**Discussion by patricktsanght** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#83] [1.0 Getting started on the Data Challenge Platform - https://datascience.ey.com/]
Hello, I am glad that this challenge introduced me to Data Cube, it would be very helpful for my future studies.
And if I want to load different dataset (e.g. Modis, Landsat), is there some resources that can help me do that?
Thank you so much!!

**Comment by EY-glowglowglow** [comment]
That's good to hear! With regards to your question, I'd start with this - https://docs.dea.ga.gov.au/notebooks/Beginners_guide/04_Loading_data.html 
@patrickmuston1 would you have anything else to add on?

**Reply by patricktsanght** [reply]
Hello, thank you so much for your reply!
What I want to know is the step before that.
How we populate the data cube, setup the yaml file, where to find for example the sentinel satellite data within certain dates in certain locations...etc.
